test_command_line_tool.d

```diff
@@ -12,13 +12,35 @@ int main(string[] args)
     try { return tryMain(args); }
     catch(SilentException) { return 1; }
 }
+
 int tryMain(string[] args)
 {
-    foreach (entry; dirEntries("test", "*.har", SpanMode.shallow))
+    // TODO: move to std.path
+    version(Windows)
+        string exeExtention=".exe";
+    else
+        string exeExtention;
+
+    auto rootDir=__FILE_FULL_PATH__.dirName;
+    auto outDir = rootDir.buildPath("out");
+    auto harExe = outDir.buildPath("har"~exeExtention);
+
+    // TODO: move this pattern to std.file using RAII
+    auto oldDir=getcwd;
+    scope(exit)
+        chdir(oldDir);
+    chdir(rootDir);
+
+    auto testDir=rootDir.buildPath("test"); // workaround https://issues.dlang.org/show_bug.cgi?id=6138 : we need absolutePath
+    auto outTestDir=outDir.buildPath("test");
+    mkdirRecurse(outTestDir);
+    foreach (entry; dirEntries(testDir, "*.har", SpanMode.shallow))
     {
-        run(format("./har %s", entry.name));
-        auto expected = entry.name.setExtension(".expected");
-        auto actual = entry.name.setExtension("");
+        auto file = entry.name.relativePath;
```

Comment:
`.relativePath` may not be needed if you make the other changes.

Reply:
done